### ISSUE TITLE
Could you create new pip release with latest changes?

### ISSUE THREAD

**Issue by dbbert** [sunscrapers/djoser#655]
The latest pip release of djoser (2.1.0) dates from October 2020, but there have been quite some changes in this repo since then. Would it be possible to create a new pip release please?

**Comment by dethos** [comment]
It would be nice.

**Comment by DBlek** [comment]
Any comment on that @dekoza Or should we use the version connected to the specific commit?

**Comment by henrique-pimenta** [comment]
@haxoza A new pip release would be much appreciated.

**Comment by spockNinja** [comment]
Yes, please. Running into issues that have been fixed, but not released. 🙏🏼  Is there anything we can do to help?
